pytest-selenium/pages/content.py

```diff
@@ -50,6 +57,30 @@ def attribution(self):
     def section_url_within_attribution(self):
         return self.find_element(*self._section_url_locator)
 
+    def scroll_over_content_overlay(self, x, y):
```

Comment:
since this function is very specifically for the content overlay, why are x and y not calculated within the function?